apps/site/components/Filter/FilterSliderUsage.tsx

```diff
@@ -0,0 +1,129 @@
+import { useSearch } from '@faststore/sdk'
+import {
+  FacetBoolean as UIFacetBoolean,
+  FacetBooleanItem as UIFacetBooleanItem,
+  FacetRange as UIFacetRange,
+  Facets as UIFacets,
+  Filter as UIFilter,
+  FilterSlider as UIFilterSlider
+} from '@faststore/ui'
+import { useFormattedPrice } from '../utilities/usePriceFormatter'
+
+import type { useFilter } from './useFilter'
+
+interface Props {
+  /**
+   * The array that represents the details of every facet.
+   */
+  facets: any[]
```

Comment:
Could you type minimally the facet? Can be just with  __typename and label

Reply:
Nice! [done](https://github.com/vtex/faststore/pull/1687/commits/ed33d1d34e240588e5272e9531cdfddde7b9007e).